spec/factories/grader_permissions.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :grader_permission, class: GraderPermission do
+    association :user, factory: :ta
```

Comment:
change this to `association :ta` when you make the changes to the association in the model